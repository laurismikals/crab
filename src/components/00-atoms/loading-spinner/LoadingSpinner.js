import React, { PureComponent } from 'react';

import './LoadingSpinner.css';

export class LoadingSpinner extends PureComponent {
  render() {
    return (
      <div className="loading"/>
    );
  }
}

export default LoadingSpinner;
