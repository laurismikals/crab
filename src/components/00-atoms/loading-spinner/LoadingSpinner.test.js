import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import { LoadingSpinner } from './LoadingSpinner';

describe('Atom: LoadingSpinner', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <LoadingSpinner />
    );
    expect(tree).toMatchSnapshot();
  });
});
