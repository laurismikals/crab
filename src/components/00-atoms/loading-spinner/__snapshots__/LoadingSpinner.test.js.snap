// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom: LoadingSpinner renders correctly 1`] = `
<div
  className="loading"
/>
`;
